My Portfolio Project: Setup

alexpuente0 opened the pull request:
In this project the following actions were taken:

- Started My Portfolio project
- Used Flexbox to layout different elements
- Made a Toolbar on top of the page
- Made a Headline section to introduce myself

alexpuente0 committed:
Create linters.yml

Created the linters folder and file to test code before merge

alexpuente0 committed:
Create index.html

Created index.html file and added the base of the page

alexpuente0 committed:
Add Image Folder

Added image folder with the icons and background to be used on the page

alexpuente0 committed:
Create styles.css

Created CSS file and styled the page using flexbox

alexpuente0 committed:
Fix Linters

-Removed comment about the fonts
-Corrected the <P> element
-Added ALT attributes to images

smunozmo reviewed (changes requested):
Hi @alexpuente0, 👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request


- [ ] You have submitted a project, that doesn't have a descriptive README file which is very important for people to know what is your project about, please update it using this [template](https://github.com/microverseinc/readme-template), this will make your repository look more professional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

alexpuente0 committed:
Update README file

Populated README file with the following information:
-Technology used
-Information about the Author
-Acknowledgments

alexpuente0 committed:
Update styles.css

Modified navigation bar height. Updated colors of the text in the navigation bar,  title, subtitle, and paragraph.

alexpuente0 committed:
Update styles.css

Modified hex code of the white color to comply with linters comment.

AmaduKamara commented on index.html:
```diff
@@ -0,0 +1,49 @@
+  </head>
+
+  <body>
+    <nav class="top-menu">
+      <p class="toolbarlogo">Jhon Doe</p>
+      <img class="toolbar menu" src="./icons/Enabled.png" alt="collapsed menu icon"/>
+    </nav>
```
[Required Changes]
- [x]  Please place your navbar in a `<header>` tag that serves as the navigation bar section
- [x] Also, please replace the `p` tag holding the (John Doe) logo with an `a` tag and place your name instead of the one given in the template. Except if you're John Doe 😂, you can leave it, else, please change it to your name.

alexpuente0 replied:
Made these changes, will keep working on the rest. Thank you Amadu!

AmaduKamara commented on index.html:
```diff
@@ -0,0 +1,49 @@
+      <img class="toolbar menu" src="./icons/Enabled.png" alt="collapsed menu icon"/>
+    </nav>
+
+    <div class="welcomeinfo">
+      <h1 class="title">Hey There. I’m Alex Puente</h1>
+      <h3 class="subtitle">I am a Software Developer</h3>
+      <div class="paragraph">
+        I can help you build a product, feature or website. Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </div>
+    </div>
+<div class="contact-bar">
+    <img class="contactbar-img icon" src="./icons/github.png" alt="github logo" />
+    <img class="contactbar-img icon" src="./icons/linkedin.png" alt="linkedin logo" />
+    <img class="contactbar-img icon" src="./icons/peace.png" alt="peace logo" />
+    <img class="contactbar-img icon" src="./icons/twitter.png" alt="twitter logo"/>
+    <img class="contactbar-img icon" src="./icons/them.png" alt="them logo"/>
+</div>  
```
[Required Changes]
- [x]  Would you please wrap this whole section in a `section` tag to better structure your code with semantic tags.
- [x] Also the `div` at line `41` holding the social icons should be placed in a `ul` tag with `li` items and each li item should be in an `a` tag to make it a link. Please work on that. 👍

alexpuente0 replied:
Just finished on these two issues and added functionality to a couple of the logos. Thank you for your great feedback

AmaduKamara commented on styles.css:
```diff
@@ -0,0 +1,67 @@
+  color: #fff;
+}
+
+.contact-bar {
+  display: flex;
+  margin-top: -180px;
+  margin-left: 80px;
+}
```
[Required Changes]
- [x] Please remove the margin-left style here so the social icons are placed on the left. That way your design will match the one given in the Figma design.

### Figma Design
![image](https://user-images.githubusercontent.com/50941074/158849622-964f0c70-ad92-4ae1-9555-03b101c25c7c.png)

### Your Design
![image](https://user-images.githubusercontent.com/50941074/158849932-61448a35-b745-4fe7-81ae-525a66be4914.png)

alexpuente0 replied:
Just finished this change and will submit it for a new review. Thank you for your help, Amadu!

AmaduKamara reviewed (changes requested):
# STATUS: REQIRED CHANGES ♻️♻️♻️

Hi @alexpuente0 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Linter checks passed
✔️ Professional README
✔️ Good commit messages

## Required Changes ♻️

- [x] Please implement all the changes marked as [Required Changes] in the PR comments

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

alexpuente0 committed:
Update index.html

-Placed navbar Inside a Header tag
-Changed the P tag in the navbar logo to an anchor tag.
-Wrapped all social icons on a section

alexpuente0 committed:
Update index.html

-Modified contact icons into an unordered list and put them anchor tags.
-Added functionality to Twitter, Linkedin, and GitHub logos.
-Formatted code for better readability.

alexpuente0 committed:
Update styles.css

Removed margin-left for the social icons so they are leftbound.

MeqdamAlqudah commented on index.html:
```diff
@@ -0,0 +1,95 @@
+          you need coded, don’t hestiate to contact me.
+        </div>
+      </div>
+      <ul class="contact-bar">
+        <li>
+          <a href="http://www.github.com">
+            <img
+              class="contactbar-img icon"
+              src="./icons/github.png"
+              alt="github logo"
+            />
+          </a>
+        </li>
+        <li>
+          <a href="http://www.linkedin.com">
+            <img
+              class="contactbar-img icon"
+              src="./icons/linkedin.png"
+              alt="linkedin logo"
+            />
+          </a>
+        </li>
```
[REQUIRED] Please make sure to put list style here to none so it looks exactly like the Figma template
[REQUIRED] As the last reviewer mentioned fix the left margin beside your icons list
![image](https://user-images.githubusercontent.com/74744787/158874384-4ced519a-176f-4661-99be-41698af98108.png)

alexpuente0 replied:
Hello Meqdam! I'm already working on it!
I've already changed the list style to none to eliminate the bullets.
I have a question about the icons margin, since on my last review I was asked to remove the margin so the icons moved to the left, so I need to add margin again?

![image](https://user-images.githubusercontent.com/99830261/158876987-170690d1-4a53-4df6-84bc-806647d39f81.png)

Thank you for your feedback!

MeqdamAlqudah commented on styles.css:
```diff
@@ -0,0 +1,66 @@
+  margin: 0;
+}
+
+.menu {
+  width: 18px;
+  height: 20px;
+}
+
+.top-menu {
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 25px;
+  width: 375px;
+  height: 12px;
+}
+
+.toolbarlogo {
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  color: #fff;
+  background-color: #3c3a39;
+  height: 20px;
+}
```
- [ ] Fix the distance between your headlines and toolbar here so it matches exactly the template
![image](https://user-images.githubusercontent.com/74744787/158875023-680fefef-2edf-4b3f-8721-abc9169d9a67.png)


![image](https://user-images.githubusercontent.com/74744787/158875092-8aaa5b86-f1ec-4a60-8c05-737b9d4ba2bb.png)

MeqdamAlqudah reviewed (changes requested):
Hi  @alexpuente0 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

![almost there](https://media.giphy.com/media/7JsChhLbpj0rJtzimC/giphy.gif)


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

alexpuente0 committed:
Update index.html

Repositioned the contact bar icons inside the div class welcome info.

alexpuente0 committed:
Update styles.css

Modified title, subtitle, and paragraph font sizes and colors.
repositioned flex containers background image and contact info icons to be more leftbound.

alexpuente0 committed:
Update index.html

Added link to make the Angel and Medium icons functional.

alexpuente0 committed:
Update styles.css

removed spaces to comply with linters

alexpuente0 committed:
Modify headline section and background image

AmaduKamara reviewed (approved):
# STATUS: APPROVED 🟢🟢🟢
Hey @alexpuente0, 

Good job implementing the changes requested by the previous reviewers.
Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

![ClappingLeonardoDicaprioGIF](https://user-images.githubusercontent.com/50941074/158897174-e2cd3185-90f8-4b41-9d16-da1ce9fc19c0.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
